queries/javascript/textobjects.scm

```diff
@@ -38,3 +38,11 @@
 
 ;; blocks
 (_ (statement_block) @block.inner) @block.outer
+
+;; parameters
+(formal_parameters
+  (identifier) @parameter.inner)
```

Comment:
this can also be an assignment_pattern (optional parameter) or object_pattern array_pattern ...

Reply:
Ohh, right. Thanks for double checking!

Should we be explicit here or just go with `(_)`?

```
(formal_parameters
  (_) @parameter.inner)
```

vs

```
;; parameters
(formal_parameters
  (identifier) @parameter.inner)

; function (v = default_value)
(formal_parameters
  (assignment_pattern) @parameter.inner)

; function ({ x }) ...
(formal_parameters
  (object_pattern) @paremeter.inner)

; function ([ x ]) ...
(formal_parameters
  (array_pattern) @paremeter.inner)

;; arguments
(arguments
  (_) @parameter.inner)
```

Thoughts?

Reply:
(pushed the latter, but let me know if you prefer the former)

Reply:
Should be fine. Thanks.